app/assets/javascripts/tylium/pages/activities.js

```diff
@@ -29,4 +29,20 @@ document.addEventListener('turbolinks:load', function() {
       }
     });
   }
+
+  $(function() {
+    $('[data-behavior~=activity-filters] select').change(function () {
+      $(this).closest('form').submit();
+    });
+
+    $('[data-behavior~=activity-filters] input[type="date"]').change(function () {
+      const dateValue = $(this).val();
+      const periodStart = $("#period_start").attr("min");
```

Comment:
We use data-attributes for [selectors in JS](https://github.com/dradis/dradis-ce/wiki/Style-Guide:-JavaScript#code) to keep things clean and clear. Looking at the view file, you may not think that changing an `id` attribute will break js functionality. Modifying a `data-behavior` attribute will likely make you think about JS.